code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -583,6 +584,10 @@
 			if(Pred && src.loc)
 				m_type = 1
 				playsound(src.loc, 'sound/voice/pred_overhere.ogg', 25, 0)
+		if ("queenstomp")
+			if(Pred && src.loc)
+				m_type = 1
+				playsound(src.loc, pick('sound/effects/alien_footstep_large1.ogg', 'sound/effects/alien_footstep_large2.ogg', 'sound/effects/alien_footstep_large3.ogg'), 35, rand(20000, 25000), 11, falloff = 4)
```

Comment:
you can just use "alien_footstep_large" here without need for a pick() because of this in sound.dm `			if("alien_footstep_large")
				S = pick('sound/effects/alien_footstep_large1.ogg','sound/effects/alien_footstep_large2.ogg','sound/effects/alien_footstep_large3.ogg')`